pm/sender.go

```diff
@@ -166,6 +166,10 @@ func (s *sender) validateTicketParams(ticketParams *TicketParams, numTickets int
 		return nil
 	}
 
+	if ev.Cmp(new(big.Rat).SetInt(ticketParams.FaceValue)) >= 0 {
+		return fmt.Errorf("total faceValue too low faceValue=%v", ticketParams.FaceValue)
```

Comment:
```suggestion
		return fmt.Errorf("ticket faceValue too low faceValue=%v", ticketParams.FaceValue)
```

^ Since we're just checking the face value provided via `ticketParams` and not a sum.